test/unit/connect-inject-clusterrole.bats

```diff
@@ -59,7 +59,7 @@ load _helpers
       --set 'global.enablePodSecurityPolicies=false' \
       . | tee /dev/stderr |
       yq -r '.rules | length' | tee /dev/stderr)
-  [ "${actual}" = "1" ]
+  [ "${actual}" = "2" ]
```

Comment:
could we use map/select for these tests? I think it'll both make it more resilient against future changes and more readable.

Reply:
sure thing, great idea!